src/main/java/duke/taskmanager/TaskList.java

```diff
@@ -0,0 +1,72 @@
+package duke.taskmanager;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    List<Tasks> lst;
```

Comment:
would it be better to add some kind of access modifiers to these variables? similar to above